src/server/controller/cycleData/updateOriginalDataPointDependentNodes.ts

```diff
@@ -53,7 +53,7 @@ export const updateOriginalDataPointDependentNodes = async (
     const data = await getTableData(
       {
         aggregate: false,
-        columns: [colName],
```

Comment:
any reason for removing column ?

Reply:
Returning empty object when columns is added (need another issue to fix this)

Reply:
dont worry. I dont think its very important now